voucher/filters.py

```diff
@@ -4,8 +4,9 @@
 from django.utils.translation import ugettext_lazy as _
 from django.utils.timezone import get_current_timezone
 
-from core.models import Card
-from voucher.models import UseLog, Wallet, WorkLog
+from core.models import NfcCard
+from voucher.models import UseLog, Wallet, WorkLog, VoucherWallet, CoffeeWallet, VoucherUseLog, CoffeeUseLog, \
```

Comment:
Går nok helt fint, men sikkert nyttig for oss begge å bli påminnet en gang i blant :p